ImprovedAnvil/lang/English/crom.tra

```diff
@@ -238,3 +238,7 @@
 @637 = ~Bring me an Elven Stone of Arcana, an Elven Stone of Holiness, a War Hammer +3, Elven Holy Water, a Permanency Scroll, and 75,000 gold and I can forge a mighty hammer for a Winged Elf.  Shall we?~
 @638 = ~What can you do with this Sword of Chaos +2?~
 @639 = ~Eh, ye hold a weakened and evil blade.  I'm afraid there is nothing I can do for ye.~
+@640 = ~What can you do with Cloak of Dragomir?~
+@641 = ~Sadly, I can be of no help. Track down a fellow by name of Maheer. An eccentric man, truth be told to ye. But they say he has a knack for handling delicate material such as one that ye have.~
+@642 = ~What can you do with this Enhanced Cloak of Dragomir?~
+@643 = ~Sadly, I can be of no help. Track down a fellow by name of Maheer. An eccentric man, truth be told to ye. But they say he has a knack for handling delicate material such as one that ye have.~

```

Comment:
This is purely cosmetic, but maybe the second line shouldn't be copied verbatim. Also, if you do make them identical, there's no need to duplicate the string, just reference @641 both times.

Reply:
updated the second line which seems preferable to keep the indexing consistent